core/themes/seven/template.php

```diff
@@ -33,6 +33,11 @@ function seven_preprocess_layout(&$variables) {
       }
     }
   }
+  $path_array = explode('/', current_path());
+  if ($path_array[0] == 'admin' && $path_array[1] == 'config' && isset($path_array[2])) {
+    $variables['classes'][] = 'admin-config';
+    $variables['classes'][] = 'admin-config-' . $path_array[2];
+  }
```

Comment:
These classes don't seem necessary, and are being added in a strange place anyway.

Reply:
removed.